File: app/src/main/java/com/seniorlibs/app/grammar/GrammarActivity.java
package com.seniorlibs.app.grammar;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

import com.seniorlibs.app.R;

public class GrammarActivity extends AppCompatActivity {

    private static final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_grammar);

        Father father = new Son();
        father.talking();
    }
}

File: kotlinlib/src/main/java/com/read/kotlinlib/grammar/GrammarActivity.kt
package com.read.kotlinlib.grammar

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.read.kotlinlib.R

class GrammarActivity : AppCompatActivity() {

    companion object {
        private const val TAG = "MainActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_grammar)
        val father: Father = Son()
        father.talking()
    }
}